src/simulation/m_hyperelastic.fpp

```diff
@@ -112,9 +112,9 @@ contains
 
         !$acc parallel loop collapse(3) gang vector default(present) private(alpha_K, alpha_rho_K, &
         !$acc rho, gamma, pi_inf, qv, G, Re, tensora, tensorb)
-        do l = 0, p
-            do k = 0, n
-                do j = 0, m
+        do l = 0, p - 2
```

Comment:
where does the -2 come from? is this `fd_order - 2 = 2`? if so, `fd_order refers to something used in post-processing and probes, not in the simulation code for finite differencing during simulation (though I would understand the confusion).

Reply:
This is due to boundary conditions that have not been coded for RMT and connected to the `s_populate_variables_buffers` issue below.. I will investigate this further.

Reply:
can you put something in `m_checker` then that stops MFC when it detects that hyperelasticity is used with a BC it does not support?

Reply:
Yes, I can definitely do that.